modules/java_api/src/test/java/org/intel/openvino/ModelTests.java

```diff
@@ -44,6 +44,20 @@ public void testGetShape() {
         assertArrayEquals("Shape", ref, outputs.get(0).get_shape());
     }
 
+    @Test
+    public void testGetPartialShape() {
+        ArrayList<Output> outputs = net.outputs();
+        int[] ref = new int[] {1, 10};
+
+        PartialShape partialShape = outputs.get(0).get_partial_shape();
+        for (int i = 0; i < ref.length; i++) {
+            Dimension dim = partialShape.get_dimension(i);
```

Comment:
@likholat we aren't allocating memory in Demension class but I found method for delete memory
Could you provide more information about it?
https://github.com/openvinotoolkit/openvino_contrib/blob/b0826c595d60616a7253382e3b04a4023f73b803/modules/java_api/src/main/java/org/intel/openvino/Dimension.java#L28
https://github.com/openvinotoolkit/openvino_contrib/blob/b0826c595d60616a7253382e3b04a4023f73b803/modules/java_api/src/main/cpp/dimension.cpp#L24
https://github.com/openvinotoolkit/openvino_contrib/blob/b0826c595d60616a7253382e3b04a4023f73b803/modules/java_api/src/main/cpp/openvino_java.hpp#L110